src/PostList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Post } from './Post';
+import { Types } from './types';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
You have li tags, but don't have ul or ol

Reply:
Oops)